dts/xtensa/intel/intel_adsp_ace15_mtpm.dtsi

```diff
@@ -83,6 +83,12 @@
 		zephyr,memory-region = "IMR1";
 	};
 
+	ssp_ports: ssp-ports {
+		compatible = "intel,ssp-ports";
+		base = <6>;
+		extended = <0>;
+	};
+
```

Comment:
Leftover from previous revision? This should be ssp-settings now

Reply:
Oops!